tests/test_browser.py

```diff
@@ -2943,7 +2943,8 @@ def test_meminit_pairs(self):
     args = ["-O2", "--memory-init-file", "0", "-s", "MEM_INIT_METHOD=2", "-s", "ASSERTIONS=1"]
     self.btest(d, expected='0', args=args + ["--closure", "0"])
     self.btest(d, expected='0', args=args + ["--closure", "0", "-g"])
-    self.btest(d, expected='0', args=args + ["--closure", "1"])
+    if not WINDOWS: # Closure compiler fails on Windows when it receives the unexpected characters in memory string initializer.
```

Comment:
My suspect at the time was that it would be a Closure bug, but I did not dig deep enough to understand and confirm. I'm now running through the `asm2m` suite to see what comes up, so I'll see after that if this is still an issue.